lib/legacy/polymer.dom.js

```diff
@@ -395,6 +395,10 @@ if (window['ShadyDOM'] && window['ShadyDOM']['inUse'] && window['ShadyDOM']['noP
     }
   });
 
+  forwardReadOnlyProperties(Wrapper.prototype, [
```

Comment:
Can you add some backstory in a comment about why this appears as such a special case?

It's not a made-up `Polymer.dom` thing, but it also never needed a Shady DOM patch?  Why was it on 1.x's `Polymer.dom` then?

Reply:
Done.